00-Starter-Seed/README.md

```diff
@@ -20,4 +27,35 @@ Once you've set those 2 enviroment variables:
 2. Start the server with `python server.py`
 3. Try calling [http://localhost:3001/ping](http://localhost:3001/ping)
 
-You can then try to do a GET to [http://localhost:3001/secured/ping](http://localhost:3001/secured/ping) which will throw an error if you don't send an access token signed with RS256 with the appropriate issuer and audience in the Authorization header. You can also try to  do a GET to [http://localhost:3001/secured/private/ping](http://localhost:3001/secured/private/ping) which will throw an error if you don't send an access token with the scope `read:agenda` signed with RS256 with the appropriate issuer and audience in the Authorization header.
+# Running the example with Docker
+
+In order to run the example you need to have `docker` installed.
```

Comment:
It's not 100% necessary to have `docker` installed in order to run the sample, so this is a little bit confusing to read. The instructions are also redundant because most of it was already explained previously. 

Proposal:

```
# Running the sample with Docker

In order to run the sample with [Docker](<link-to-docker's-website>) you need to add the `AUTH0_DOMAIN` and `API_ID` to the `.env` filed as explained [previously](<link-to-previous-instructions-if-possible>) and then  

1. Execute `sh exec.sh` in the command line to run the Docker in Linux, or `.\exec.ps1` to run the Docker in Windows.
2. Try calling [http://localhost:3001/ping](http://localhost:3001/ping)
```
I would also move this section to the bottom of the README because it's an extra step only needed for those using Docker